mtls-southbound/apiproxy/SecureBackend-v1.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<APIProxy revision="5" name="SecureBackend-v1">
+    <DisplayName></DisplayName>
```

Comment:
Remove all other attributes and just have the Description, please refer to other samples

Reply:
No one cares which properties are in this file, but fine.